client.py

```diff
@@ -0,0 +1,74 @@
+import tkinter as tk
+import requests
+
+
+def main():
+    """
+    run interactive tkinter envinroment as client
+    """
```

Comment:
Single line titles can be 1 line.
```suggestion
    """ run interactive tkinter envinroment as client """
```